openslides/motions/views.py

```diff
@@ -66,6 +66,16 @@ def check_view_permissions(self):
             result = False
         return result
 
+    def retrieve(self, request, *args, **kwargs):
+        """
+        Customized view endpoint to retrieve a motion.
+
+        Adds the allowed actions for the motion.
+        """
+        response = super().retrieve(request, *args, **kwargs)
+        response.data['allowed_actions'] = self.get_object().get_allowed_actions(request.user)
+        return response
```

Comment:
I can live with this method, if someone creates an issue, that this function has to be recreated in javascript and deleted from this place